gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapperTest.java

```diff
@@ -238,24 +253,48 @@ public void doesntMergeObservationWhereTheMatchingObservationHasMultiplePertinen
     public void doesntMergeObservationWhereTheObservationDoesNotEndInEllipses() {
         var ehrExtract = createExtract(List.of(
                 createObservation("ID-1", "101", "This is an observation doesnt end with ellipses:::"),
-                createObservation("ID-2", "101", "This is an observation which ends with ellipses removed."),
+                createObservation("ID-2", "102", "This is an observation which ends with ellipses removed."),
```

Comment:
Still a bit unclear as to why this line was changed.

Reply:
it can be changed to its original value as this change was made under the previous acceptance criteria which were changed recently...